toolchain/defs.bzl

```diff
@@ -326,21 +326,21 @@ _SYSROOTS = {
 }
 
 _TOOLCHAINS = {
-    "10.3.0": {
+    "12.3.0": {
```

Comment:
Instead of replacing, the idea is to add another entry to the _TOOLCHAINS dictionary.